### ISSUE TITLE
[Schema Inaccuracy] Fix media types in rest/gists/comments.md

### ISSUE THREAD

**Issue by cmwilson21** [github/rest-api-description#1904]
Transferring an OS PR over. - https://github.com/github/docs/pull/22245

The media types seem to be wrong on the page https://docs.github.com/en/rest/gists/comments

To test the current ones, which seem to have no effect:

```
$ curl --silent -H "Accept: application/vnd.github.3.base64+json" https://api.github.com/gists/9257657/comments/1221189 | grep body 
  "body": "Thanks, Octocat!\n"
```

To test the new ones, which have an effect:

```
$ curl --silent -H "Accept: application/vnd.github.3.full+json" https://api.github.com/gists/9257657/comments/1221189 | grep body 
  "body_html": "<p dir=\"auto\">Thanks, Octocat!</p>",
  "body_text": "Thanks, Octocat!",
  "body": "Thanks, Octocat!\n"
```

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

**Comment by nickfloyd** [comment]
Please let us know if this work is still needed.  We're trying to follow up on any older issues to make sure they get properly routed.

**Comment by becco** [comment]
Closing as stale.

**Comment by maflcko** [comment]
Yeah, this is still an issue. The initial steps to reproduce still work: The docs claim there to be a "base64" encoding, but there is none.

```
$ date --utc && curl --silent -H "Accept: application/vnd.github.base64+json" https://api.github.com/gists/9257657/comments/1221189 | grep body 
Thu  8 Jan 19:20:52 UTC 2026
  "body": "Thanks, Octocat!\n"
```

However, there are the other media types (see above for the reproducer).

So either the docs will have to be fixed, or the backend.
